main.tf

```diff
@@ -137,13 +147,13 @@ module "nfs" {
 
   count                          = var.storage_type == "standard" ? 1 : 0
   name                           = "${var.prefix}-nfs"
-  azure_rg_name                  = module.resource_group.name
+  azure_rg_name                  = local.aks_rg.name
   azure_rg_location              = var.location
   proximity_placement_group_id   = element(coalescelist(azurerm_proximity_placement_group.proximity.*.id, [""]), 0)
   vnet_subnet_id                 = module.vnet.subnets["misc"].id
   machine_type                   = var.nfs_vm_machine_type
-  azure_nsg_id                   = module.nsg.id
-  tags                           = module.resource_group.tags
+  azure_nsg_id                   = local.nsg.id
+  tags                           = var.tags
```

Comment:
This seems as though it was set in the module and now it's a full blown var. Can you clarify?

Reply:
not sure what "it" refers to here.

Reply:
It was the module.resource_group but you answered the vars questions above.